src/main/java/de/mediathekview/mserver/crawler/basic/AbstractCrawler.java

```diff
@@ -123,7 +126,15 @@ public synchronized MServerBasicConfigDTO getCrawlerConfig() {
   public synchronized MServerConfigDTO getRuntimeConfig() {
     return runtimeConfig;
   }
+  
+  public JsoupConnection getConnection() {
```

Comment:
In den ganzen crawler Implementierungen  geißt es immer `crwaler.getConnection().getString`. Neben dem, dass ich `getString` recht nichts sagend finde, siehe Comment in der `JsoupConnection` könnte man das auch abkürzen und direkt ein `requestBody`oder so in dieser Klasse anbieten. Entwder zusätzlich oder anstatt dem `getConnection` je nach dem ob man die connection noch für irgend etwas anderes braucht.

Reply:
shortcuts mit gleichem Namen eingerichtet